charts/gardener/charts/utils-common/templates/_secret-alerting.yaml

```diff
@@ -0,0 +1,27 @@
+{{- define "gardener.secret-alerting-smtp" -}}
+{{- if .Values.global.controller.enabled }}
+{{- range $key, $config := .Values.global.controller.alerting }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: alerting-smtp-{{ $key }}
```

Comment:
```suggestion
  name: alerting-{{ $key }}
```

Reply:
Changing the secret name will require manual intervention (deleting the old secret) if gardener deployed the alerting-smtp secret in the past. This is why I did not change the name or make the other keys configurable.

Reply:
I guess since the label is changing it will not be a problem and then Gardener can delete the secret with the old label. I will implement this.

Reply:
Hm, shouldn't Helm clean up the old secret?

Reply:
Yeah you're right. I'll add the new keys to the secret.